lib/vsc/utils/fancylogger.py

```diff
@@ -345,6 +345,8 @@ def deprecated(self, msg, cur_ver, max_ver, depth=2, exception=None, *args, **kw
             self.raiseException("DEPRECATED (since v%s) functionality used: %s" % (max_ver, msg), exception=exception)
         else:
             deprecation_msg = "Deprecated functionality, will no longer work in v%s: %s" % (max_ver, msg)
+            if verbose:
+                sys.stderr.write(deprecation_msg)
```

Comment:
what's the point of this? too lazy to grep logfile? logging shouldn't mess with stderr directly. setup another handler that uses stderr and redirect all error and warnings to it if you really want this